src/staging/src/templates/staticLanding1.js

```diff
@@ -70,9 +70,9 @@ const LandingPage = ({ pageContext }) => {
   return (
     <>
       <Seo
-        title={meta.metaTitle}
-        description={meta.description}
-        keywords={meta.metaKeywords}
+        title={meta?.metaTitle || pageContext.page.Title}
```

Comment:
I think that should be page?.Title based on nearby examples

Reply:
Thanks. page?.title wasn't working so I dug this up. I missed the capital 'T'.